pipeline/src/aggregator/mod.rs

```diff
@@ -3124,7 +3124,7 @@ mod tests {
             | 1                 | baeabeibrydyefwuzizf6s32lbuu27rpjibrbtsziqukscbnuo3y52pgmha | 2           | did:key:bob   | {controller: 6469643a6b65793a626f62, model: ce01040171710b0009686d6f64656c2d7631}                                                                   | baeabeibrydyefwuzizf6s32lbuu27rpjibrbtsziqukscbnuo3y52pgmha | 0          | 0            | {"content":{"accountRelation":{"fields":["creator"],"type":"set"},"implements":[],"interface":false,"name":"TestSmallModel","relations":{},"schema":{"$schema":"https://json-schema.org/draft/2020-12/schema","additionalProperties":false,"properties":{"blue":{"format":"int32","type":"integer"},"creator":{"type":"string"},"green":{"format":"int32","type":"integer"},"red":{"format":"int32","type":"integer"}},"required":["creator","red","green","blue"],"title":"SmallModel","type":"object"},"version":"2.0","views":{}},"metadata":{}} | []                                                        |
-            | 2                 | baeabeihrpnqmppjqdohl2stvfevu7jn57zjqzqb4dflbl4ettogmt56xoy | 3           | did:key:alice | {controller: 6469643a6b65793a616c696365, model: ce01020100122031c0f042da99464be96f4b0d29afc5e9406219cb2885152105b476f1dd3ccc38, unique: 616c696365} | baeabeials2i6o2ppkj55kfbh7r2fzc73r2esohqfivekpag553lyc7f6bi | 0          | 0            | {"content":null,"metadata":{}}                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                      | []                                                        |
+            | 2                 | baeabeihrpnqmppjqdohl2stvfevu7jn57zjqzqb4dflbl4ettogmt56xoy | 3           | did:key:alice | {controller: 6469643a6b65793a616c696365, model: ce01020100122031c0f042da99464be96f4b0d29afc5e9406219cb2885152105b476f1dd3ccc38, unique: 616c696365} | baeabeials2i6o2ppkj55kfbh7r2fzc73r2esohqfivekpag553lyc7f6bi | 0          | 0            | {"content":null,"metadata":{}}                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                      | [Missing content for SET account relation]                |
```

Comment:
I changed this behavior.. this should be inline with js-ceramic behavior that rejects data events for SET account relation without content.

Reply:
also added a new test for a mismatch, so the diff looks wonky but it adds a new event 2 and moves everything down